### PR TITLE
aliexpress.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [X] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/139559

### Add your comment and screenshots

If you cannot reproduce, please change your delivery destination to `United States`.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/allowlist.txt [hunk 1]
```diff
@@ -37,6 +37,8 @@
 !#endif
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/139559
+@@||aliexpress.*^$cookie=/^_ga_/
```
Comment (Alex-302) [inline SpywareFilter/sections/allowlist.txt:41]: How to reproduce it? I can't with `.com` and `.us`

Comment (piquark6046) [reply SpywareFilter/sections/allowlist.txt:41]: > How to reproduce it?

Please read `Add your comment and screenshots`.

Comment (Alex-302) [reply SpywareFilter/sections/allowlist.txt:41]: I get this error (even without AdGuard)

<details>
<summary>Details</summary>

image

</details>

@jellizaveta @Sergey-Lyapin please take a look.

**Review by zloyden — APPROVED** [review]

File: SpywareFilter/sections/allowlist.txt (hunk 1 above)
Comment (zloyden) [reply SpywareFilter/sections/allowlist.txt:41]: I have the same error and the rule above helps.

**Pull request merged**